src/Sarif.UnitTests/Readers/UriConverterTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+using FluentAssertions;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.Readers
+{
+    public class UriConverterTests
+    {
+        [Theory]
```

Comment:
I had no idea that VS would execute both XUnit tests and MSTest tests in the same assembly.